### PR TITLE
Fix wording

### PR DESCRIPTION
**Pull request by sprankhub**
## Purpose of this pull request

This pull request (PR) fixes the wording for the `magentoDataFixtureBeforeTransaction` annotation. Fixtures only support PHP script files, but no classes.

## Affected pages

- https://developer.adobe.com/commerce/testing/guide/integration/annotations/

### CODE REVIEW

**Comment by sprankhub** [comment]
Reading through https://developer.adobe.com/commerce/testing/guide/integration/attributes/data-fixture-before-transaction/, I am not sure if my PR is correct :D Maybe it is indeed possible to use classes by now? Then we should maybe make this "to a class, a script file or a method"?

**Review by jeff-matthews — COMMENTED** [review]
Thanks for the PR @sprankhub. I'm confirming with internal engineers.

**Review by thiaramus — APPROVED** [review]

File: src/pages/guide/integration/annotations/index.md [hunk 1]
```diff
@@ -25,8 +25,8 @@ Admin Configuration Fixture|`@magentoAdminConfigFixture`|`@magentoAdminConfigFix
...
-Data Fixture|`@magentoDataFixture`|`@magentoDataFixture <script_filename> / <method_name>/<fully_qualified_class_name> [as:alias with:{}]`|Points to a class or a method which creates testing entities (fixtures) for test execution. These are applied during the transaction.
-Data Fixture Before Transaction|`@magentoDataFixtureBeforeTransaction`|`@magentoDataFixtureBeforeTransaction <script_filename>/<method_name>`|Points to a class or a method which creates testing entities (fixtures) for test execution before the transaction has begun. You will need to implement a rollback file for changes made here. (e.g. Fixture file my_fixture.php would also require a my_fixture_rollback.php that reverts the original fixture's changed.)
+Data Fixture|`@magentoDataFixture`|`@magentoDataFixture <script_filename> / <method_name>/<fully_qualified_class_name> [as:alias with:{}]`|Points to a file or a method which creates testing entities (fixtures) for test execution. These are applied during the transaction.
```
Comment (thiaramus) [inline src/pages/guide/integration/annotations/index.md:28]:
```suggestion
Data Fixture|`@magentoDataFixture`|`@magentoDataFixture <script_filename> / <method_name> [as:alias with:{}]`|Points to a file or a method which creates testing entities (fixtures) for test execution. These are applied during the transaction.
```

**Review by thiaramus — CHANGES_REQUESTED** [review]

**Comment by jeff-matthews** [comment]
@sprankhub, please see/accept the suggestion from @thiaramus and we'll get this merged.

**Review by jeff-matthews — APPROVED** [review]

**Pull request merged**